### PR TITLE
deploy in dev with TLS enabled

### PR DESCRIPTION
**Pull request by xcoulon**
URL will be `https://...` but with an invalid
certificate.

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: deploy/deployment_dev.yaml [hunk 1]
```diff
@@ -56,4 +56,6 @@ spec:
     kind: Service
     name: registration-service
     weight: 100
+  tls:
+    termination: edge
```
Comment (alexeykazakov) [inline deploy/deployment_dev.yaml]: It didn't pass the linter check:
```
yamllint -c .yamllint ./deploy/deployment_dev.yaml ./deploy/role.yaml ./deploy/role_binding.yaml ./deploy/service_account.yaml
./deploy/deployment_dev.yaml
  60:22     error    trailing spaces  (trailing-spaces)

make: *** [lint-yaml] Error 1
```
```suggestion
    termination: edge
```

Comment (alexeykazakov) [reply deploy/deployment_dev.yaml]: Fixed in https://github.com/codeready-toolchain/registration-service/pull/57/commits/f99aab9fcb9d08120e92a5b6ac05362f8cb111f9

Comment (xcoulon) [reply deploy/deployment_dev.yaml]: oh sorry :/ thanks for fixing this issue, @alexeykazakov !

**Comment by xcoulon** [comment]
thanks for the review, @tinakurian and @alexeykazakov 🙌

**Pull request merged**